src/shared/components/NotFound.tsx

```diff
@@ -136,10 +140,23 @@ const NotFound: FC = () => {
 
   const handleDeepLink = useCallback(async () => {
     if (!org.current) {
-      setIsFetchingOrg(true)
-      org.current = await fetchOrg()
-    }
+      if (shouldUseQuartzIdentity()) {
+        try {
+          setIsFetchingOrg(true)
+          const defaultAccount = await getDefaultAccount()
+          const quartzOrgs = await fetchOrgsByAccountID(defaultAccount.id)
 
+          const deafultQuartzOrg =
+            quartzOrgs.find(org => org.isDefault) || quartzOrgs[0]
+          org.current = deafultQuartzOrg
+        } catch (error) {
+          console.error(error)
```

Comment:
We're trying to move away from using `console.error` anywhere in code directly.

I'm not 100% certain on this, but I think if we reach this point, we might want to redirect the user to the no orgs page. Doing something like [this](https://github.com/influxdata/ui/blob/master/src/shared/containers/SetOrg.tsx#L116) is probably what we want to do here.

Reply:
great idea. Yeah, routing user to `/no-orgs` page makes sense if for some reason our api fails to fetch org id.